app/Http/Controllers/DesktopAuthController.php

```
@@ -0,0 +1,106 @@
+            'locked_user_id' => $lockedUserId,
+        ]);
+    }
+
+    #[Post('/setup', name: 'desktop.setup', middleware: ['guest', 'throttle:desktop-auth'])]
+    public function setup(Request $request, SetupDesktopAdmin $setupDesktopAdmin): RedirectResponse
+    {
+        abort_unless(User::query()->doesntExist(), 404);
+
+        $user = $setupDesktopAdmin->create($request->all());
+
+        Auth::login($user);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Auth' in method 'setup'. (undefined)

(StaticAccess)